lib/web_ui/dev/test_runner.dart

```diff
@@ -381,7 +379,7 @@ class TestCommand extends Command<bool> with ArgUtils {
     );
 
     if (exitCode != 0) {
-      throw ToolException(
```

Comment:
Can you please remove the name change. Let's discuss name changes on these files (and also location refactors) before you start a PR.

Reply:
Done. I don't mind discussing code changes in the PR. The reason for rename was [consistency with flutter/flutter](https://github.com/flutter/flutter/blob/b24bca1383fa8553e5a39a831fb19ee214ef233f/packages/flutter_tools/lib/src/base/common.dart#L21), and because we don't use it as an exception (e.g. we don't care about the stack trace).